ypathgen/pathgen.go

```diff
@@ -262,6 +266,21 @@ func (cg *GenConfig) GeneratePathCode(yangFiles, includePaths []string) (*Genera
 		errs = util.AppendErrs(errs, es)
 	}
 
+	namer := func(dir *ygen.Directory, dirs map[string]*ygen.Directory) string {
```

Comment:
I'm assuming this will be made into a function?

Reply:
fixed